src/source/PeerConnection/Rtp.c

```diff
@@ -76,6 +127,11 @@ STATUS freeKvsRtpTransceiver(PKvsRtpTransceiver* ppKvsRtpTransceiver)
     if (pKvsRtpTransceiver->sender.retransmitter != NULL) {
         freeRetransmitter(&pKvsRtpTransceiver->sender.retransmitter);
     }
+
+    if (pKvsRtpTransceiver->pRollingBufferConfig != NULL) {
```

Comment:
```
if (pKvsRtpTransceiver->pRollingBufferConfig != NULL) {
```
Isn't this check unnecessary since SAFE_MEMFREE of freeRollingBufferConfig checks for null already?

The pre-requisite of this function would be pKvsRtpTransceiver != NULL, which is already checked above on line 114